Zend/zend_builtin_functions.c

```diff
@@ -979,20 +971,41 @@ ZEND_FUNCTION(property_exists)
 	}
 	RETURN_FALSE;
 }
+
+/* {{{ Checks if the object or class has a property */
+ZEND_FUNCTION(property_exists)
```

Comment:
To reduce repetition, is it realistic to write a macro like:

```
#define ZEND_FUNCTION_CALL_FRAMELESS(name, minArgs, maxArgs) \
ZEND_FUNCTION(name) { \
    uint32_t args = ZEND_NUM_ARGS(); \
    if (args > maxArgs || args < minArgs) { /* handle arg count mismatch */ } \
    switch (args) { \
        /* here, do some macro magic to only compile calls between min and maxargs */ \
        case 2: ZEND_FRAMELESS_FUNCTION_NAME(name, 2)(return_value, EX_VAR(0), EX_VAR(1)); break;
    } \
}
```

Making it more effortless and straightforward. Also probably avoids compiling the body of the function twice (not requiring an extra inline function too).